e2etest/webhookstests/webhooks.go

```diff
@@ -381,3 +383,175 @@ var _ = ginkgo.Describe("Webhooks", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Webhooks", func() {
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("Clearing any previous configuration")
+		err := ConfigUpdaterOtherNS.Clean()
+		framework.ExpectNoError(err)
+	})
+
+	ginkgo.AfterEach(func() {
+		if ginkgo.CurrentGinkgoTestDescription().Failed {
+			k8s.DumpInfo(Reporter, ginkgo.CurrentGinkgoTestDescription().TestText)
+		}
+
+		// Clean previous configuration.
+		err := ConfigUpdaterOtherNS.Clean()
+		framework.ExpectNoError(err)
+	})
+
+	ginkgo.Context("For BFDProfile", func() {
+		ginkgo.It("Should reject creating BFDProfile in a different namespace", func() {
```

Comment:
I think you could use a Table test here, providing resources as a table entry. This will avoid duplication, since we always expect the same behaviour.
See https://github.com/metallb/metallb/blob/65a54f3a97eb1a5d34a3671b187c3cd09fa65bae/e2etest/bgptests/bgp.go#L152 for example